layouts/partials/main-menu-mobile.html

```diff
@@ -2,7 +2,9 @@
   <ul>
     {{ $currentPage := . }}
     {{ range .Site.Menus.main }}
-    <li class="menu-item-{{ .Name | lower }}{{ if $currentPage.IsMenuCurrent "main" . }} active{{ end }}">
+    {{ $active := or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }}
```

Comment:
Do we need this line? The other menus dont use it. If checking the relLanUrl is enough we should remove this line right?

Reply:
It seems to me like [`.IsMenuCurrent`](https://gohugo.io/functions/ismenucurrent/) *should* be enough for setting the `active` class on pages – including the home page –, based on its description in the docs and the examples at https://gohugo.io/templates/menu-templates/. With that in mind, I would consider that The Right Way of doing things and the line below Just A Workaround. Following that reasoning, I think it's reasonable to keep both lines. What do you think?

Whether `.HasMenuCurrent` should be kept I guess depends on how nested menus are dealt with by the theme and what behavior is desired. The way this template is currently implemented, it seems to me that nested menus are not supported and `.HasMenuCurrent` thus redundant.

Reply:
That reasoning makes sense to me. It makes things more robust but the ideal approach is stated first.

I don't mind if we leave `.HasMenuCurrent`